include/openssl/bn.h

```diff
@@ -337,6 +337,7 @@ BIGNUM *BN_mod_sqrt(BIGNUM *ret,
                     const BIGNUM *a, const BIGNUM *n, BN_CTX *ctx);
 
 void BN_consttime_swap(BN_ULONG swap, BIGNUM *a, BIGNUM *b, int nwords);
+void BN_consttime_swap_randomized(BN_ULONG swap, BIGNUM *a, BIGNUM *b, int nwords, BN_ULONG rand1, BN_ULONG rand2);
```

Comment:
This new public function needs to be documented. You also need to run "make update" to update the util/libcrypto.num file.

Reply:
I'd like to see a test for this new function

Reply:
For a new PR, what would a test of this function look like?